.tools/psalm/baseline-taint.xml

```diff
@@ -1,144 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <files psalm-version="4.4.1@9fd7a7d885b3a216cff8dec9d8c21a132f275224">
   <file src="redaxo/src/addons/structure/plugins/history/fragments/history/layer.php">
-    <TaintedHtml occurrences="56">
+    <TaintedHtml occurrences="4">
       <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1iframe')</code>
-      <code>$this-&gt;getVar('content1select')</code>
-      <code>$this-&gt;getVar('content1select')</code>
```

Comment:
Dass die baseline-taint hier so stark schrumpft, scheint mir korrekt zu sein.
Bei den Fragmenten ist das Problem, dass die `setVar()`-Aufrufe nicht den korrekten Fragmenten zugeordnet werden können, sondern wahllos mit mehreren `getVar()`-Aufrufen innerhalb von Fragmenten verknüpft werden.

Daher taucht eine problematische Eingabe in `setVar` dann hier gleich mehrfach auf, und fällt somit nun auch mehrfach weg.

So jedenfalls meine Analyse. 100%ig verstehe ich die baseline natürlich auch nicht.